frontend/packages/dev-console/src/components/topology/TopologyApplicationList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+import { ResourceLink } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+export type TopologyApplicationResourceListProps = {
+  resources: K8sResourceKind[];
+};
+
+export type ApplicationResourceListItemProps = {
+  resource: K8sResourceKind;
+};
+
+const ApplicationResourceListItem: React.FC<ApplicationResourceListItemProps> = ({
+  resource: {
+    metadata: { name, namespace },
+    kind,
+  },
+}) => {
+  return (
+    <li className="list-group-item  container-fluid">
+      <ResourceLink kind={kind} name={name} namespace={namespace} />
+    </li>
+  );
+};
+
+const TopologyApplicationResourceList: React.FC<TopologyApplicationResourceListProps> = ({
```

Comment:
We should prefer having one component per file. I think in this case you can simply move the `<li>` markup into the `<ListGroup>` markup since that component is not being used anywhere and seems simple enough to me.